zabbix-monitoring-cluster/nested/monitoringServer.json

```diff
@@ -0,0 +1,245 @@
+        "description": "The existing vnet name the monitor VM connects to."
+      }
+    },
+    "existingSubnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "The existing subnet name the monitor VM connects to, also agents will also be installed on those VMs."
+      },
+      "defaultValue": "default"
+    },
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name for the monitor vm."
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "metadata": {
+        "description": "Name for the monitor vm."
+      },
+      "defaultValue": "Standard_DS2"
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Monitor VM user name."
+      },
+      "defaultValue": "monitoruser"
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Monitor VM password."
+      }
+    },
+    "mysqlHost": {
+      "type": "string",
+      "metadata": {
+        "description": "Monitoring service backend MySQL host. Using 'localhost' here will set up a new MySQL instance on the monitor VM."
+      },
+      "defaultValue": "localhost"
+    },
+    "mysqlDbName": {
+      "type": "string",
+      "metadata": {
+        "description": "Monitoring service backend MySQL database name."
+      },
+      "defaultValue": "zabbixdb"
+    },
+    "mysqlUser": {
+      "type": "string",
+      "metadata": {
+        "description": "Monitoring service backend MySQL user name."
+      },
+      "defaultValue": "zabbixuser"
+    },
+    "mysqlPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Monitoring service backend MySQL password."
+      }
+    },
+    "smtpServer": {
+      "type": "string",
+      "metadata": {
+        "description": "Smtp server, format is server:port."
+      },
+      "defaultValue": ""
+    },
+    "smtpUser": {
+      "type": "string",
+      "metadata": {
+        "description": "Smtp username."
+      },
+      "defaultValue": ""
+    },
+    "smtpPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Smtp password."
+      },
+      "defaultValue": ""
+    },
+    "notificationReceiver": {
+      "type": "string",
+      "metadata": {
+        "description": "Notification receiver's Email."
+      },
+      "defaultValue": ""
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/zabbix-monitoring-cluster"
+    }
+  },
+  "variables": {
+    "location": "[tolower(resourceGroup().location)]",
```

Comment:
any particular reasion you tolower the resourceGroup().location? It should work as-is without tolower